src/Components/Common/utils/Tooltip.tsx

```diff
@@ -0,0 +1,30 @@
+import clsx from "clsx";
+import { ReactNode, useState } from "react"
+
+export default function ToolTip(props : {
+    children : ReactNode
+    text : ReactNode
+    position : "TOP" | "BOTTOM" | "LEFT" | "RIGHT"
```

Comment:
You can make it as an optional prop and use `ToolTip.defaultProps = { position : "TOP" } ` for default value